Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed case ISX-1436 (UI TK Input Action Asset Editor - Error deleting Bindings with DeleteKey on Windows).
 - Fixed issue with UI Toolkit based Input Action Editor not restoring it's selected items after Domain Reload.
 - Fixed the [`GetHapticCapabilitiesCommand`](xref:UnityEngine.InputSystem.XR.Haptics.GetHapticCapabilitiesCommand) always failing to execute due to a mismatch in the size in bytes of the payload and the size expected by XR devices. Changed [`HapticCapabilities`](xref:UnityEngine.InputSystem.XR.Haptics.HapticCapabilities) to include all properties returned by the XR input subsystem. This makes Input System match the functionality provided by the [XR](https://docs.unity3d.com/Manual/com.unity.modules.xr.html) module's [`InputDevice.TryGetHapticCapabilities`](https://docs.unity3d.com/ScriptReference/XR.InputDevice.TryGetHapticCapabilities.html) and [`HapticCapabilities`](https://docs.unity3d.com/ScriptReference/XR.HapticCapabilities.html).
+- Fixed case ISX-1680 (UITK Input Action Asset Editor - Deleting a binding selects a seemingly random binding)
```

Comment:
We typically try to avoid referencing internal ticket IDs since they make little sense to users since not publicly available while the release notes and the repo is. Note that ISXB tickets are referenced via public issue tracker though. I would rather suggest explaining the fixed scenario rather than reusing the ticket title, e.g.
"Fixed an issue where deleting a binding in the Input Action Editor while some binding items where not expanded would result in an unexpected selection change that may expand unrelated bindings in the tree view."

Reply:
Updated CHANGELOG entry with fbacf6f958456ecc949d89a22634f77574cfc607.